src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs

```diff
@@ -633,3 +633,7 @@
   {:path "/profile/billing"
    :method :get
    :mock-data (utils/rand-subset ["broad-dsde-dev" "broad-institute"])})
+
+(defn submissions-queue-status []
+  {:path "/submissions/queueStatus"
+   :method :get })
```

Comment:
It's just some fake data in the same format as the real data.  When you go into fake data mode, the endpoint calls return it instead of actually making the ajax call.  Mock data for this endpoint would look very similar to `get-entity-types` on line 220.